drawBot/drawBotDrawingTools.py

```diff
@@ -2369,14 +2369,18 @@ def ImageObject(self, path=None):
         """
         return self._imageClass(path)
 
-    def Variable(self, variables, workSpace):
+    def Variable(self, variables, workSpace, isContinuous=True):
         """
         Build small UI for variables in a script.
 
         The `workSpace` is usually `globals()`
         as you want to insert the variable in the current workspace.
         It is required that `workSpace` is a `dict` object.
 
+        Setting `isContinuous` to `False` will add a "Update" button at the bottom.
+        for the user to click to get and update. The default is `True` and will
+        set all control updating the script continuously.
```

Comment:
This needs editing. Perhaps:
"""
The `isContinuous` argument controls whether whether script is run when UI elements change. The default is `True`, which will execute the script immediately and continuously when the user input changes. When set to `False`, there will be an "Update" button added at the bottom of the window. The user will have to click this button to execute the script and see the changes. This is useful when the script is slow, and continuous execution would decrease responsiveness.
"""

Reply:
Btw. I think I would still prefer to call it `continuous`, to be more consistent with Vanilla.

Reply:
changed!